assets/styles/styles.css

```diff
@@ -0,0 +1,347 @@
+  transform: translateY(-100%);
+}
+
+/* typography */
+.sticky-nav,
+.main-header {
+  background-color: var(--nav-bg-color);
+  font-size: 18px;
+  line-height: 1.67;
+}
+
+.main-header {
+  min-height: 100px;
+  z-index: 3;
+  position: relative;
+}
+
+.sticky-nav {
+  position: sticky;
+  top: 0;
+  z-index: 5;
+}
+
+.sticky-nav i {
+  color: #ff999a;
+  font-size: 26px;
+}
+
+.navbar-2 {
+  font-size: 1.1875rem;
+  line-height: 1.263;
+}
+
+.navbar-3 {
+  font-size: 1rem;
+  line-height: 1;
+  color: var(--header-label-color);
+}
+
+.header-label-text {
+  font-size: 16px;
+  line-height: 1;
+  color: var(--header-label-color);
+  font-weight: 700;
+  margin-top: 20px;
+}
+
+.header-main-heading {
+  font-size: 1.375rem;
+  line-height: 1.09;
+  font-weight: 700;
+}
+
+.otherSections-main-heading {
+  font-size: 1.25rem;
+  line-height: 1;
+  font-weight: 700;
+}
+
+.EP-aside-heading {
+  font-size: 1.0625rem;
+  line-height: 1.23;
+  font-weight: 700;
+}
+
+.p-text {
+  font-family: 'Arial', sans-serif;
+  font-size: 0.9375rem;
+  line-height: 1.3333;
+  color: var(--p-color);
+}
+
+.my-turn-text {
+  color: var(--header-label-color);
+  font-size: 1rem;
+  line-height: 1.1875;
+}
+
+.small-text {
+  font-family: '.Helvetica Neue Interface', sans-serif;
+  font-size: 0.9075rem;
+  line-height: 1.4;
+  color: var(--header-label-color);
+}
+
+.btn {
+  color: white;
+  background-color: var(--main-theme-color);
+  font-size: 1.125rem;
+  line-height: 1;
+}
+
+.btn:hover {
+  color: white;
+}
+
+.nav-btn {
+  background-color: #ffe43f;
+  border-radius: 500px;
+}
+
+/* end typography */
+
+/* nav */
+.X:first-child {
+  margin-left: 10px;
+}
+
+.wrapper {
+  max-width: 1300px;
+  margin: 0 auto;
+}
+
+.moreS-card-wrapper::before {
+  content: '';
+  display: block;
+  flex-direction: column;
+  border-top: 0.5px solid #ddd;
+  width: 94%;
+  margin: 10px auto auto;
+}
+
+.moreS-card img {
+  width: 100px;
+  height: 100px;
+}
+
+.opinion-card img,
+.debate img {
+  width: 63px;
+}
+
+.ePick-c3 img {
+  width: 95px;
+}
+
+.opinion-card p,
+.debate p {
+  color: var(--opinion-author-text-color);
+  font-size: 0.875rem;
+  line-height: 1.23;
+}
+
+.moreS-contentBox p {
+  font-size: 0.8125rem;
+  line-height: 1.38;
+  color: var(--p-color);
+}
+
+.moreS-contentBox h6 {
+  font-size: 0.9444rem;
+  line-height: 1.17;
+}
+
+.debate {
+  border-top: 3px double var(--header-label-color);
+  border-bottom: 3px double var(--header-label-color);
+}
+
+.debate-middle {
+  border-top: 3px double #d9d9d9;
+}
+
+.debate-middle p {
+  padding: 7px;
+  left: 50%;
+  top: -12px;
+}
+
+.inTheMag-cardLabel p,
+.section-label p,
+.E-PICK-subHead {
+  color: var(--main-theme-color);
+  cursor: pointer;
+}
+
+.inTheMag-cardLabel span {
+  background-color: var(--main-theme-color);
+  cursor: default;
+}
+
+.covers img {
+  height: 227px;
+}
+
+#subscribe-cover2 {
+  left: 43%;
+  top: 7%;
+}
+
+.newsletter img {
+  /* height: 300px;
+    width: 200px; */
```

Comment:
- [ ] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /* `.nav { color: red;}` */  
:heavy_check_mark: **comments about code** /*Navigation */